daemon/transmission-daemon.service

```diff
@@ -8,6 +8,8 @@ Type=notify
 ExecStart=/usr/bin/transmission-daemon -f --log-error
 ExecReload=/bin/kill -s HUP $MAINPID
 NoNewPrivileges=true
+ProtectSystem=full
+PrivateTmp=true
```

Comment:
What are the particular files daemon creates in /tmp that you think need protection, and from who?

Reply:
The idea is that if transmission creates anything in `/tmp` other users won't be able to read it regardless of the permissions transmission sets on it:

```
drwx------ 3 jelly   users     60 Apr 28 12:49 systemd-private-4fcbe22b8820420e844f57e4d7fe07a3-transmission.service-59b7yP
```